lib/ts/recipe/emailpassword/components/features/signInAndUp/index.tsx

```diff
@@ -132,6 +133,7 @@ export function useChildProps(
                 rid: recipe!.config.recipeId,
                 successRedirectContext: {
                     action: "SUCCESS",
+                    isNewPrimaryUser: true,
```

Comment:
this is not necessary since a sign up can lead to account linking to an existing user. So you need to actually check the number of items in the login method, and if thats 1, only then this is true, else it's false.

Reply:
Applies to all recipes.

Reply:
done